src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search-form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="search-form__input search-form__input-big"
```

Comment:
```suggestion
        class="search-form__input"
```
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/07af3501-d41a-4ba8-bbe0-abf6658a6efc)

Reply:
made first input as basic